features/step_definitions/prie_outcome_steps.rb

```diff
@@ -0,0 +1,94 @@
+Given('user bulk loaded a file with PRIE outcomes') do
```

Comment:
What do you think about
```suggestion
Given('user bulk loads monthly statement with reconsideration hearing (PRIE) outcomes') do
```